boards/arm/lpcxpresso55s69/lpcxpresso55s69.dtsi

```diff
@@ -181,20 +181,6 @@ mikrobus_serial: &flexcomm2 {
 mikrobus_spi: &hs_lspi {
 };
 
-&dma0 {
-	/*
-	 * The total number of dma channels available is defined by
-	 * FSL_FEATURE_DMA_NUMBER_OF_CHANNELS in the SoC features file.
-	 * Since memory from the heap pool is allocated based on the number
-	 * of DMA channels, set this property to as many channels is needed
-	 * for the platform. Adjust HEAP_MEM_POOL_SIZE in case you need more
-	 * memory.
-	 */
-	dma-channels = <20>;
-	nxp,dma-num-of-otrigs = <4>;
-	status = "okay";
-};
```

Comment:
Do we want to delete everything here or just the portion with `dma-channels`

Reply:
it is all being moved to other files